client/.eslintrc.json

```diff
@@ -10,6 +10,7 @@
     "plugin:@typescript-eslint/recommended"
   ],
   "overrides": [],
+  "ignorePatterns": ["**/*.json"],
```

Comment:
Why are we ignoring JSON files? Wouldn't it be great to lint those as well?

Reply:
Base linter seemed to not understand our translations.json. I've added a plugin for linting JSON instead and removed the ignored pattern.